homeassistant/components/vallox/__init__.py

```diff
@@ -183,6 +200,22 @@ async def async_set_profile_fan_speed_boost(
             return False
         return True
 
+    async def async_set_profile(
+        self, profile: str, duration: int | None = None
+    ) -> bool:
+        """Activate profile for given duration."""
+        _LOGGER.debug("Activating profile %s for %s min", profile, duration)
+        try:
+            await self._client.set_profile(
+                PRESET_MODE_TO_VALLOX_PROFILE[profile], duration
+            )
+        except ValloxApiException as err:
+            _LOGGER.error(
+                "Error setting profile %d for duration %s: %s", profile, duration, err
+            )
+            return False
+        return True
```

Comment:
I assume the true false is something that is for vallox?

Reply:
I just imitated the other code in the vallox module, maybe someone else can answer

Reply:
Hmm, not 100% sure but might just be a leftover from #56966.